Containers/talk/Dockerfile

```diff
@@ -46,6 +46,7 @@ COPY --chmod=775 healthcheck.sh /healthcheck.sh
 COPY --chmod=664 supervisord.conf /supervisord.conf
```

Comment:
and in the janus build as well?

Reply:
The preinstalled packages aren't statucally linked during build, so I think it isn't needed

Reply:
ok